pkgs/tools/security/pcsctools/default.nix

```diff
@@ -13,20 +14,26 @@
 
 stdenv.mkDerivation rec {
   pname = "pcsc-tools";
-  version = "1.6.0";
+  version = "1.6.2";
 
-  src = fetchurl {
-    url = "http://ludovic.rousseau.free.fr/softwares/pcsc-tools/${pname}-${version}.tar.bz2";
-    sha256 = "sha256-ZRyN10vLM9tMFpNc5dgN0apusgup1cS5YxoJgybvi58=";
+  src = fetchFromGitHub {
+    owner = "LudovicRousseau";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-c7md8m1llvz0EQqA0qY4aGb3guGFoj+8uS4hUTzie5o=";
   };
 
+  preConfigure = ''
+    ./bootstrap
+  '';
+
```

Comment:
This is exactly what `autoreconfHook` does for you, no need to do it explicitly.

Reply:
Thanks tired that and didn't worked I suspect because of the wrong buildInputs I wasn't aware of.